src/utils.ts

```diff
@@ -683,3 +683,13 @@ export function lexicographicCompare(a: string, b: string): number {
     // hidden the operation in this function.
     return (a < b) ? -1 : ((a === b) ? 0 : 1);
 }
+
+const collator = new Intl.Collator();
+/**
+ * Performant language-sensitive string comparison
+ * @param a the first string to compare
+ * @param b the second string to compare
+ */
+export function compare(a: string, b: string): number {
+    return collator.compare(a, b);
+}
```

Comment:
kind of annoying that we're duplicating this in react-sdk now, can you open a maintenance issue to use the js-sdk util instead of duplicating along with merging this please